Vagrantfile

```diff
@@ -38,7 +38,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     pg1.vm.provision "shell", inline: "systemctl restart network"
     pg1.vm.provision "shell", inline: "yum clean all"
     pg1.vm.provision "shell", inline: "puppet apply /vagrant/scripts/local-hosts.pp"
-    pg1.vm.provision "shell", inline: "puppet module install puppetlabs-concat --version 2.2.1" #Dependency requirement of theforeman-puppet
```

Comment:
That was something I manually added for a brief time because puppetlabs-puppetdb requiring a newer version of concat, while theforeman-puppet module required an older version and would cause `vagrant up` to fail to run. No longer needed since the requirements were bumped up in theforeman-puppet.